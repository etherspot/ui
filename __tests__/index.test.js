// index.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import TestModule from '../src/index';

test('TestModule renders correctly', () => {
  const tree = renderer.create(<TestModule />).toJSON();
  expect(tree).toMatchSnapshot();
});