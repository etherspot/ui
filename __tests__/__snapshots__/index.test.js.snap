// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestModule renders correctly 1`] = `
<div>
  Hello Etherspot UI
</div>
`;
